src/extensions/default/DebugCommands/main.js

```diff
@@ -48,6 +51,13 @@ define(function (require, exports, module) {
     
     var KeyboardPrefs = JSON.parse(require("text!keyboard.json"));
     
+    // default preferences file name
+    var DEFAULT_PREFERENCES_FILENAME = "defaultPreferences.json",
+        SUPPORTED_PREFERENCE_TYPES   = ["number", "boolean", "string", "array", "object"];
+
+    var reComputeDefaultPrefs        = true,
```

Comment:
I will change this.I guess I was using `computeDefaultPrefs` before and kind of blindly added a `re`.